disquaire_project/manager/models.py

```diff
@@ -20,11 +20,13 @@ def __str__(self):
         return self.name
 
 class Disk(models.Model):
+    # There is no unicity constraint and no index on the reference. Are you sure you need this?
```

Comment:
I aggree.
We need to add constraints and index. Is it something that has to go on the views or on the model? I though I should to a validation to check the reference is an integer and is unique.